src/PageContent/Calendar/DelivererGroupContent.js

```diff
@@ -0,0 +1,164 @@
+import React, { Component } from 'react';
+import { AccountType, StringFormat } from '../../Enums';
+import './Content.css';
+import volunteer from '../../icons/volunteer.svg';
+
+class DelivererGroupContent extends Component {
+    constructor(props) {
+        super(props);
+        if (this.props.delivery) {
+            this.state = {
+                edit: false,
+                delivererGroup: this.props.delivery.delivererGroup.name,
+                deliverer1: this.props.delivery.delivererGroup.deliverers[0]
+                    .name,
+                deliverer2: this.props.delivery.delivererGroup.deliverers[1]
+                    .name,
+                phone1: this.props.delivery.delivererGroup.deliverers[0].phone,
+                phone2: this.props.delivery.delivererGroup.deliverers[1].phone,
+                email1: this.props.delivery.delivererGroup.deliverers[0].email,
+                email2: this.props.delivery.delivererGroup.deliverers[1].email
+            };
+        } else {
+            this.state = {
+                edit: false
+            };
+        }
+
+        this.edit = this.edit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+    edit() {
+        this.setState({
+            edit: true
+        });
+    }
+
+    handleChange(e) {
+        e.preventDefault();
+
```

Comment:
```
let deliverers = [
    {
        name: e.target.deliverer1.value,
        phone: e.target.phone1.value,
        email: e.target.email1.value
    },
    {
        name: e.target.deliverer2.value,
        phone: e.target.phone2.value,
        email: e.target.email2.value
    }
];
this.setState({
    edit: false,
    deliverers: deliverers
});
```
(when we handle the backend code, we will actually get rid of `setState` for the `deliverers` and write to db instead.)